LICENSE

```diff
@@ -0,0 +1,25 @@
+BSD 2-Clause License
+
+Copyright (c) 2019, Mollie
```

Comment:
Please make this Mollie B.V. just like the readme and the composer.json

Reply:
fixed.